sessions/recap-project-2/example-implementation/form.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App</title>
+    <link rel="stylesheet" href="./styles.css" />
+    <script src="form.js" defer></script>
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header__title">Add Question</h1>
+    </header>
+    <main>
+      <form class="question-form" data-js="form">
+        <label class="question-form__input">
+          <span>Question</span>
+          <textarea
+            name="question"
+            maxlength="150"
+            data-js="question-input"
+          ></textarea>
+          <span
+            class="question-form__character-count"
+            data-js="question-character-count"
+            >0/150</span
+          >
+        </label>
+        <label class="question-form__input">
+          <span>Answer</span>
+          <textarea
+            name="answer"
+            maxlength="150"
+            data-js="answer-input"
+          ></textarea>
+          <span
+            class="question-form__character-count"
+            data-js="answer-character-count"
+            >0/150</span
+          >
+        </label>
+        <label class="question-form__input">
+          <span>Tag</span>
```

Comment:
Same a above, do we need the `span`? 🤔 

```suggestion
          Tag
```